src/components/Projects/ProjectCard/ProjectCard.tsx

```diff
@@ -37,7 +37,7 @@ const ProjectCard = ({ project, hoverable = false }: Props) => {
         {proposalInCliffPeriod ? <CliffProgress enactedAt={enacted_at} /> : <VestingProgress project={project} />}
       </div>
       <div className="ProjectCard__UpdateContainer">
-        <ProposalUpdate proposal={project} update={update} expanded={false} index={update?.index} isLinkable={false} />
+        <ProjectUpdateCard authorAddress={project.user} update={update} index={update?.index} isLinkable={false} />
```

Comment:
shouldn't this be the address of the update author? (could be a coauthor, right?)